<launch>
  <!-- Namespace for the node and services -->
  <arg name="ns"  default="gqcnn" />
  
  <!-- Configuration file for Grasp Planner Node -->
  <arg name="config"           default="$(find gqcnn)/cfg/ros_nodes/kinect2_client_node.yaml" />
  <arg name="world_camera_tf"           default="$(find gqcnn)/data/calib/kincet2_world_identity.tf" />
  <arg name="model_dir"        default="$(find gqcnn)/models/GQ-Suction" />
  
  <node name="kinect2_client"  pkg="gqcnn" type="kinect2_client_node.py" output="screen" >
    <param name="config"           value="$(arg config)" />
    <param name="world_camera_tf"  value="$(arg world_camera_tf)" />
    <param name="model_dir"        value="$(arg model_dir)" />
  </node>
</launch>